source/_integrations/youtube.markdown

```diff
@@ -0,0 +1,68 @@
+---
+title: YouTube
+description: Instructions on how to integrate YouTube within Home Assistant.
+ha_category:
+  - Sensor
+ha_release: '2023.6'
+ha_iot_class: Cloud Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@joostlek'
+ha_domain: youtube
+ha_platforms:
+  - sensor
+ha_integration_type: service
+---
+
+The YouTube integration allows you to connect YouTube channels to Home Assistant.
+
+## Prerequisites
+
+You need to configure developer credentials to allow Home Assistant to access your Google Account.
+These credentials are the same as the ones for [Nest](/integrations/nest) and [Google Sheets](/integrations/google_sheets).
```

Comment:
Let's also mention Google Mail. All 3 can use the same app credentials as Youtube. Maybe we also want to include a section for unblocking popups in case people might open an issue for not getting a window in the web browser. This sounds silly but it can happen and will happen as Home Assistant becomes more and more mainstream.